packages/hydrogen/src/customer/types.ts

```diff
@@ -68,9 +71,12 @@ export type CustomerAccount = {
       RawGqlString
     >
   ) => Promise<
-    CustomerAPIResponse<
-      ClientReturn<CustomerAccountQueries, RawGqlString, OverrideReturnType>
-    >
+    Omit<
+      CustomerAPIResponse<
+        ClientReturn<CustomerAccountQueries, RawGqlString, OverrideReturnType>
+      >,
+      'errors'
+    > & {errors: JsonGraphQLError[]}
```

Comment:
Afaik, the `errors` property returned by the SFAPI at least can be undefined. Not sure about CAAPI but I assume it's the same?

```suggestion
    > & {errors?: JsonGraphQLError[]}
```

Reply:
I marked it as non optional because the original type is non optional. but this make sense too